dask_geopandas/core.py

```diff
@@ -402,15 +403,17 @@ def geohash(self, string=True, p=12):
         ----------
         as_string : bool (default True)
```

Comment:
The docstring says ``as_string``, but the signature has `string=True/False`. Does somebody remember which of the two we prefer?

(`as_string` might be a bit more explicit?)

Reply:
I like `as_string` more but don't remember the discussion.